composer.json

```diff
@@ -60,7 +60,7 @@
         "doctrine/doctrine-cache-bundle": "~1.0",
         "mmoreram/simple-doctrine-mapping": "~0.1",
 
-        "sebastian/money": "~1.3",
+        "sebastian/money": "1.5.*",
```

Comment:
I know. Just saying that if you put `1.5.*` then you cannot use `1.6.*`, and this is not the behavior we want.

If you want this change to be merged, please use the `~` symbol -> `~1.5`